composer.json

```diff
@@ -14,12 +14,20 @@
     "php": "^8.3",
     "ext-gd": "*",
     "ext-dom": "*",
-    "sculpin/sculpin": "3.3.0-alpha3",
-    "symfony/process": "^6.4|^7.0"
+    "sculpin/sculpin": "dev-main",
+    "symfony/process": "^6.4|^7.0",
+    "tempest/highlight": "^2.7",
+    "league/commonmark": "^2.4.2"
   },
   "require-dev": {
     "roave/security-advisories": "dev-latest"
   },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/pekhov14/sculpin"
```

Comment:
Do you plan on maintaining this fork? I don't think it's a good idea for the website to depend on "dev-main" on a third party composer repo like this. If you intend to maintain this fork, register it under a different vendor name on packagist.org? Or better yet get this merged into sculpin itself? Either way I don't think this reference to a third party repo overwriting sculpin is a good idea to merge into the php foundation website.

Reply:
@naderman  Yes, I made a pull request and it was accepted, I will update the code soon.